vscode/okteto.yml

```diff
@@ -1,7 +1,7 @@
 name: vote
 image: okteto/vote:ssh
 workdir: /src
-command: [ "sh", "-c", "cp /etc/ssh-key/* /root/.ssh && /usr/sbin/sshd -D" ]
+command: [ "sh", "-c", "env > /root/.ssh/environment && cp /etc/ssh-key/* /root/.ssh && /usr/sbin/sshd -D" ]
```

Comment:
Interesting. I'd assume this was automatic. Are these the variables defined in the deployment manifest?

Reply:
Yes, everyone available within the container.